optaplanner-examples/src/main/java/org/optaplanner/examples/machinereassignment/optional/score/MachineReassignmentConstraintProvider.java

```diff
@@ -189,8 +183,7 @@ protected Constraint processMoveCost(ConstraintFactory factory) {
                 .filter(processAssignment -> processAssignment.isMoved() && processAssignment.getProcessMoveCost() > 0)
                 .join(MrGlobalPenaltyInfo.class,
                         Joiners.filtering((processAssignment, penalty) -> penalty.getProcessMoveCostWeight() > 0))
-                .penalizeLong(MrConstraints.PROCESS_MOVE_COST,
-                        HardSoftLongScore.ONE_SOFT,
+                .penalize(MrConstraints.PROCESS_MOVE_COST, HardSoftLongScore.ONE_SOFT,
```

Comment:
Why using `penalize` instead of `penalizeLong` if the score is `HardSoftLongScore` ?

Reply:
Because it finally works since a couple weeks ago and we don't need to see the `implicit type cast` IDE warnings. :-)